runtime/gc_glue_java/ScavengerRootClearer.cpp

```diff
@@ -213,6 +217,54 @@ MM_ScavengerRootClearer::scavengeUnfinalizedObjects(MM_EnvironmentStandard *env)
 	env->enableHotFieldDepthCopy();
 }
 #endif /* J9VM_GC_FINALIZATION */
+
+void
+MM_ScavengerRootClearer::scavengeContinuationObjects(MM_EnvironmentStandard *env)
+{
+	MM_HeapRegionDescriptorStandard *region = NULL;
+	GC_HeapRegionIteratorStandard regionIterator(_extensions->heapRegionManager);
+	GC_Environment *gcEnv = env->getGCEnvironment();
+	bool const compressed = _extensions->compressObjectReferences();
+	while(NULL != (region = regionIterator.nextRegion())) {
```

Comment:
Missing space in 'while('. 
Please go through all (not just newly introduced) potential 'if(' and 'while(' missing-space patterns in this file and ScavengerRootScanner, and fix them.
And while there, convert remaining UDATA to uintptr_t.